✅ wandb initialized!
Epoch 1/50:   0%|                                                                  | 0/93 [01:47<?, ?it/s, loss=2.8425]Traceback (most recent call last):
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\rheehs\GIST\4-1\딥러닝\HairNet\train\train_wandb.py", line 99, in <module>
    train()
  File "C:\Users\rheehs\GIST\4-1\딥러닝\HairNet\train\train_wandb.py", line 57, in train
    loss.backward()
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\_tensor.py", line 648, in backward
    torch.autograd.backward(
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\autograd\__init__.py", line 353, in backward
    _engine_run_backward(
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\autograd\graph.py", line 824, in _engine_run_backward
    return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
KeyboardInterrupt
Traceback (most recent call last):
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\rheehs\GIST\4-1\딥러닝\HairNet\train\train_wandb.py", line 99, in <module>
    train()
  File "C:\Users\rheehs\GIST\4-1\딥러닝\HairNet\train\train_wandb.py", line 57, in train
    loss.backward()
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\_tensor.py", line 648, in backward
    torch.autograd.backward(
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\autograd\__init__.py", line 353, in backward
    _engine_run_backward(
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\torch\autograd\graph.py", line 824, in _engine_run_backward
    return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
KeyboardInterrupt
Exception ignored in atexit callback: <function _start_and_connect_service.<locals>.teardown_atexit at 0x00000245324A6CB0>
Traceback (most recent call last):
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\wandb\sdk\lib\service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\wandb\sdk\lib\service_connection.py", line 226, in teardown
    self._router.join()
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\site-packages\wandb\sdk\interface\router.py", line 75, in join
    self._thread.join()
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\threading.py", line 1096, in join
    self._wait_for_tstate_lock()
  File "C:\Users\rheehs\anaconda3\envs\HairNet\lib\threading.py", line 1116, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt:
[0m
